webdriver-spec.html

```diff
@@ -1136,6 +1136,15 @@
    e.g. attempting to click an element
    that is no longer attached to the <a>document</a>.
  </tr>
+ 
+ <tr>
+  <td><dfn>element intercepting click</dfn>
+  <td>400
+  <td><code>element intercepting click</code>
+  <td>An element <a>command</a> could not be completed
+   because there is an element in front of it intercepting
+   the command.
```

Comment:
It’s not clear to me what “intercepting the command” means. Also I think we should point out explicitly _when_ this error would occur, e.g. for which specific commands.